### ISSUE TITLE
New release soon?

### ISSUE THREAD

**Issue by otizonaizit** [mdp-toolkit/mdp-toolkit#93]
Hi,

there are some bugs that are making MDP unusable on modern systems (either with newer Pythons or with newer numpy/scipy). In particular, we need a release which contains https://github.com/mdp-toolkit/mdp-toolkit/pull/83 , which would take care of this: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1007813 . Also https://github.com/mdp-toolkit/mdp-toolkit/issues/92 needs to be fixed.

Any plans to finish the ongoing work? Or should I try to cut a release right now? @Stewori , @NiMlr, @nkgevorgyan : what's your take on this?

**Comment by Stewori** [comment]
Luckily, Nils jumped in for #92. Since #83 was merged, are we good to go then? I just had a look at #80, but do not really remember why it stalled. Probably we should leave that out as it has lower priority.

**Comment by NiMlr** [comment]
It looks like we will have to migrate from TravisCI (disabled the free tier a while ago) to (I propose) Github Actions.

**Comment by Stewori** [comment]
Yes, I just noticed the tests were not running (but I suppose you tested successfully on your computer). I never had the time and energy to look into Github Actions config and the switch from Travis. However, a friendly dev contributed recently the same kind of switch to my project pytypes: https://github.com/Stewori/pytypes/commit/aa68ed2268aaa9698cd3a55fb6155f080d8dc265
That PR can serve as a blueprint for what needs to be changed. There was one addition to the change later here: https://github.com/Stewori/pytypes/commit/6fb16a56e06f74eb4b600ed0ca74810546305b01

**Comment by otizonaizit** [comment]
hi all, any progress with the new release?

**Comment by Stewori** [comment]
@otizonaizit I cannot tackle this at the moment. If you want, please feel free to release the current state. I think the core issues were fixed. Tests are not running online ATM but as a preparation for the release you would run local tests anyway. That must do it for now.

**Comment by Debilski** [comment]
Added GitHub Actions in https://github.com/mdp-toolkit/mdp-toolkit/pull/95

**Comment by Stewori** [comment]
Thank you, @Debilski . I merged #94, so tests have a chance to pass.

**Comment by NiMlr** [comment]
There is not much happening in #80. See the [diff](https://github.com/mdp-toolkit/mdp-toolkit/pull/80/files).
It should not cause problems if merged.
